# New task's information

string taskUUID # Universally Unique Identifier (UUID)

uint64 timeStamp #The encountering time

uint16 handlingDuration

uint16 timeOutDuration

string requiredResources # Required resources for this task

